### PR TITLE
natia 12

### PR DESCRIPTION
**Pull request by putkara**

### CODE REVIEW

File: 12-PB-operators-logical/12natia.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+//0
```
Comment (Ungoot) [inline 12-PB-operators-logical/12natia.js:1]: "use strict";

File: 12-PB-operators-logical/12natia.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+//a
+console.log(x > z && x > y); 
+// expected output:false
+
+//b
+console.log(x !== y);
+// expected output:true
+
+//c
+console.log(z !== y || x != y);
```
Comment (Ungoot) [inline 12-PB-operators-logical/12natia.js:35]: This is not correct. You should use <= and >=.

File: 12-PB-operators-logical/12natia.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+// expected output:true
+
+//d
+console.log(x === z || x !== z);
```
Comment (Ungoot) [inline 12-PB-operators-logical/12natia.js:39]: x !== y

File: 12-PB-operators-logical/12natia.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+// expected output:true
+
+//e
+console.log(x * z >= 10 || x * y >= 100);
```
Comment (Ungoot) [inline 12-PB-operators-logical/12natia.js:43]: There should be no multiplication here.

Comment (Ungoot) [reply 12-PB-operators-logical/12natia.js:43]: Also it says AND, not OR

Comment (putkara) [reply 12-PB-operators-logical/12natia.js:43]: " f) x multiplied by z is"
so whot i shuld  write?

Comment (Ungoot) [reply 12-PB-operators-logical/12natia.js:43]: This is e) not f)

File: 12-PB-operators-logical/12natia.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+//a
+console.log(x > z && x > y); 
+// expected output:false
+
+//b
+console.log(x !== y);
+// expected output:true
+
+//c
+console.log(z !== y && x !== y);
```
Comment (Ungoot) [inline 12-PB-operators-logical/12natia.js:37]: c) z is less than y OR z is greater than x. You need to use < and >

File: 12-PB-operators-logical/12natia.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+// expected output:true
+
+//d
+console.log(x !== z || x !== z);
+// expected output:true
+
+//e
+console.log(x && z >= 10 || x &&y >= 100);
```
Comment (Ungoot) [inline 12-PB-operators-logical/12natia.js:45]: e) x is greater than or equal to 10 AND y is less than or equal to 10.

**Pull request closed without merging**